.github/actions/validate-docs-links/src/index.ts

```diff
@@ -9,7 +9,6 @@ import GithubSlugger from 'github-slugger'
 import matter from 'gray-matter'
 import * as github from '@actions/github'
 import { setFailed } from '@actions/core'
-import type { Node, Data } from 'unist'
```

Comment:
@delbaoliveira I'm not sure what this type was since the package doesn't exist https://www.npmjs.com/package/unist

So I fixed it by using the return type below on line 91.